src/core/Akka.API.Tests/CoreAPISpec.ApprovePersistence.approved.txt

```diff
@@ -243,6 +243,7 @@ namespace Akka.Persistence
         public void PersistAsync<TEvent>(TEvent @event, System.Action<TEvent> handler) { }
         protected abstract bool ReceiveCommand(object message);
         protected abstract bool ReceiveRecover(object message);
+        protected void RunTask(System.Func<System.Threading.Tasks.Task> action) { }
```

Comment:
Actually it is unique to Eventsourced, because of the Persist callbacks.
Non-persistence actors already have other equivalents, e.g. UntypedActor have RunTask, ReceiveActor have ReceiveAsync